metabase/metabase
# use simple_event schema to track iframe card creation

[Slack convo](https://metaboat.slack.com/archives/C064QMXEV9N/p1728927068374829)

### Description

We want to limit snowplow schema changes and utilize `simple_event` with a flexible schema more often for tracking. This PR changes the iframe cards tracking. 

Previously, the event assumed to have the following shape with its own schema:
```
{
    event: "new_iframe_card_created",
    dashboard_id: 1,
    domain_name: "example.com",
}
```

After switching to `simple_event` it will have this shape
```
{
    event: "new_iframe_card_created",
    target_id: 2,
    event_detail: "example.com",
}
```

### How to verify

An [E2E spec](https://github.com/metabase/metabase/pull/48739/files#diff-cfb3a167f618f4bcc7c0e578d6485d27a884ef0d24609e570160407d6b4e2dd1R1157) verifies correct tracking.

- Enable analytics tracking
- Create a dashboard
- While editing add iframe dashcards and save
- Ensure new_iframe_card_created event is fired with the hostname of the iframe content

### Checklist
